Makefile

```diff
@@ -1,71 +1,12 @@
-# NOTE(dkorolev):
-#
-# Yes, I am well aware it is ugly to have a `Makefile` for a `cmake`-built project.
-#
-# However, there is quite some value.
-# Please see the README of https://github.com/C5T/Current/tree/stable/cmake for more details.
-#
-# Besides, this `Makefiles` makes `:mak` in Vim work like a charm!
-
-.PHONY: release debug release_dir debug_dir release_test debug_test test fmt clean
-
-# When this `Makefile` is used to run a `cmake`-based Current build, user-defined dependencies go here.
-# No quotes needed. Space- or colon- or semicolon-separated all work fine.
-# TODO(dkorolev): Test that this works with `leveldb` too.
-C5T_DEPS=websockets
-
-# Set this var to anything non-empty to speed up builds that do not require `googletest`.
-C5T_NO_GTEST=
-
-DEBUG_BUILD_DIR=$(shell echo "$${DEBUG_BUILD_DIR:-.current_debug}")
-RELEASE_BUILD_DIR=$(shell echo "$${RELEASE_BUILD_DIR:-.current}")
-
-OS=$(shell uname)
-ifeq ($(OS),Darwin)
-  CORES=$(shell sysctl -n hw.physicalcpu)
-else
-  CORES=$(shell nproc)
-endif
-
+.PHONY: fmt run all
 CLANG_FORMAT=$(shell echo "$${CLANG_FORMAT:-clang-format}")
-
-release: release_dir CMakeLists.txt
-	@MAKEFLAGS=--no-print-directory cmake --build "${RELEASE_BUILD_DIR}" -j ${CORES}
-
-release_dir: ${RELEASE_BUILD_DIR} .gitignore
-	@grep "^${RELEASE_BUILD_DIR}/$$" .gitignore >/dev/null || echo "${RELEASE_BUILD_DIR}/" >>.gitignore
-
-${RELEASE_BUILD_DIR}: CMakeLists.txt src
-	@C5T_DEPS="${C5T_DEPS}" C5T_NO_GTEST="${C5T_NO_GTEST}" cmake -DCMAKE_BUILD_TYPE=Release -B "${RELEASE_BUILD_DIR}" .
-
-test: release
-	@(cd "${RELEASE_BUILD_DIR}"; make test)
-
-debug: debug_dir CMakeLists.txt
-	@MAKEFLAGS=--no-print-directory cmake --build "${DEBUG_BUILD_DIR}" -j ${CORES}
-
-debug_dir: ${DEBUG_BUILD_DIR} .gitignore
-	@grep "^${DEBUG_BUILD_DIR}/$$" .gitignore >/dev/null || echo "${DEBUG_BUILD_DIR}/" >>.gitignore
-
-${DEBUG_BUILD_DIR}: CMakeLists.txt src
-	@C5T_DEPS="${C5T_DEPS}" C5T_NO_GTEST="${C5T_NO_GTEST}" cmake -B "${DEBUG_BUILD_DIR}" .
-
-debug_test: debug
-	@(cd "${DEBUG_BUILD_DIR}"; make test)
-
-test: release_test
+all:
+	#pip install plsbuild
+	pls b
```

Comment:
Nit: You can do something like `pls version >/dev/null || pip install plsbuild` =)